### PR TITLE
Add GPU tests

### PR DESCRIPTION
**Pull request by akshaysridhar**
Re-introduce GPU tests. Demonstrate compatibility of higher-level function with GPUs.

### CODE REVIEW

File: test/runtests_gpu.jl [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import CLIMAParameters as CP
+
+# Create Parameters
+include(joinpath(pkgdir(SurfaceFluxes), "parameters", "create_parameters.jl"))
+
+@info "GPU Compatibility Tests"
+
+import CUDA: device
+device(::Type{T}) where {T <: CUDA.CuArray} = CUDAKernels.CUDADevice()
+ArrayType = CUDA.CuArray
```
Comment (sriharshakandala) [inline test/runtests_gpu.jl:30]: Can we print the device and array type!

Comment (akshaysridhar) [reply test/runtests_gpu.jl:29]: Updated.

**Comment by akshaysridhar** [comment]
bors try

**Comment by akshaysridhar** [comment]
bors try

**Comment by akshaysridhar** [comment]
bors +

**Pull request merged**